ecpy_hqc_legacy/instruments/drivers/visa/yokogawa.py

```diff
@@ -126,6 +127,74 @@ def voltage_range(self, v_range):
 
     @instrument_property
     @secure_communication()
+    def current(self):
+        """Current getter method. NB: does not check the current function.
```

Comment:
Docstrings should end by a blank line and the NB could be on its own line

Reply:
Ok

Reply:
NB should be moved to its own line like in the following docstring.